identity-iota/src/document/iota_document.rs

```diff
@@ -311,6 +311,7 @@ impl IotaDocument {
   }
 
   /// Remove a [`Service`] identified by the given [`IotaDIDUrl`] from the document.
+  // TODO: return an error or bool if no service was removed?
```

Comment:
I agree, then we can remove this:

https://github.com/iotaledger/identity.rs/blob/630a61e9c9aa25273b3cf5718844b3e563bd28c3/identity-account/src/updates/update.rs#L264-L267

Doesn't need to be in this PR.